README.md

```diff
@@ -16,27 +16,28 @@ pipenv install --dev
 Extract translatable text from an eQ survey with
 
 ```
-pipenv run ./scripts/run_translation_extract.sh <json_file> [output_directory]
+pipenv run python -m cli.template_extractor <schema_file> <output_directory>
 ```
-This will output the translatable text to an Excel (.xlsx) file.
+This will output the translatable text to an POT file.
+
+
+After the text has been translated, create a new translated survey with:
 
-After the text has been translated, create a new translated survey with
 ```
-pipenv run ./scripts/run_translate_survey.sh <json_file> <translations_file> [output_directory]
+pipenv run python -m cli.translate_survey <schema_file> <output_directory>
```

Comment:
this is missing the `TRANSLATION_PATH`?

`pipenv run python -m cli.translate_survey <schema_file> <translation_path> <output_directory>`

Reply:
So it is, updated.